doc/README.open_ce_test.md

```diff
@@ -0,0 +1,136 @@
+# README for `open-ce test` command
+
+* [`open-ce test env`](#open-ce-test-env-sub-command)
+* [`open-ce test feedstock`](#open-ce-test-feedstock-sub-command)
+
+## Open-CE Tests
+
+Open-CE provides tools for running tests for packages built by the Open-CE tools. To add tests to a feedstock, create a file named `tests/open-ce-tests.yaml`. The test can include a list of tests that will be run within a newly generated conda environment that should include the packages being built by the feedstock.
+
+### Open-CE Test File
+
+An Open-CE Test file is a YAML file that describes the tests that will be run. The test files can also include jinja code. Below is a simple example of a test file:
+
+```yaml
+tests:
+  - name: First Test
+    command: |
+      cd my_package
+      pytest -v tests
+{% if build_type == 'cuda' %}
+  - name: CUDA Test
+    command: |
+      cd my_package
+      pytest -v cuda_tests
+{% endif %}
+```
+
+This example will:
+
+1. Create a new conda environment that will be active throughout the running of the tests.
+1. Run the bash instructions specified by `command` in "First Test" within a new bash shell that activates the conda environment from step 1.
+1. If running within a CUDA environment, run the bash instructions specified by `command` in "CUDA Test" within a new bash shell that activates the conda environment from step 1.
+1. Removes the conda environment created in step 1.
+
+### Test Labels
+
+Labels can be added to tests using jinja. Labels can be activated using the `--test_labels` argument to the `open-ce test` tools. Below is an example of a test file that only runs the second test if the `long` label is activated.
+
+```yaml
+tests:
+  - name: Short Test
+    command: ./run_short_tests
+{% if long %}
+  - name: Long Test
+    command: ./run_long_tests
+{% endif %}
+```
+
+## `open-ce test env` sub command
+
+The `open-ce test env` command can be used to run tests for every package listed within an Open-CE Environment [file](README.yaml.md). Tests will be run for every combination of build variants that are specified. For example, if `python_versions` is set to `3.6,3.7`, tests will be run for the python 3.6 packages and python 3.7 packages.
```

Comment:
Shall we say here that the - `Tests will be run for every combination of build variants that were specified during build`.

Reply:
No, when using the `open-ce test env` command, the user can specify new build variants (in case they don't want to run tests for every variant that was built).